puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/deploy_puppet.yaml

```diff
@@ -0,0 +1,16 @@
+- job:
+    name: deploy_puppet
+    project-type: workflow
+    logrotate:
+      numToKeep: 3
+    scm:
+      - theforeman.org
```

Comment:
This doesn't sound right.

Reply:
wow i musta been a lot more tired when i put this together than i thought ...

Reply:
updated!